helpers/validateNextUsage.js

```diff
@@ -0,0 +1,22 @@
+// Ensure Next.js is available.
+// We use `peerDependencies` instead of `dependencies` so that users can choose
+// the Next.js version. However, this requires them to install "next" in their
+// site.
+const validateNextUsage = function (failBuild) {
+  if (!hasPackage('next')) {
+    return failBuild(
+      'This site does not seem to be using Next.js. Please run "npm install next" or "yarn next" in the repository.',
+    )
+  }
+}
+
+const hasPackage = function (packageName) {
+  try {
+    require(packageName)
+    return true
+  } catch (error) {
+    return false
+  }
+}
+
+module.exports = { validateNextUsage }
```

Comment:
curious why exported as object instead of just `module.exports = validateNextUsage`

Reply:
If this file exports new methods in the future, we don't need to change the places where `validateNextUsage()` was imported. 
I can change it to a default export if you prefer though. Which one do you prefer?

Reply:
no preference! just used to the default non-object export consistent throughout NoN but i'm totally cool with being future-proof here, i think it makes sense if you think we might put another func/helper in here. up to you!

Reply:
Let's go with consistency and use a default export :+1: